adapters/mqtt-vertx/src/test/java/org/eclipse/hono/adapter/mqtt/impl/VertxBasedMqttProtocolAdapterTest.java

```diff
@@ -204,6 +204,36 @@ public void testOnPublishedMessageUsesDeviceIdentityForTopicWithoutTenant(final
         assertThat(downstreamAddress.getTenantId(), is("my-tenant"));
     }
 
+    /**
+     * Verifies that the adapter supports all required topic names
+     *
+     * @param ctx The helper to use for running tests on vert.x.
+     */
+    @Test
+    public void testTopicNames(final TestContext ctx) {
```

Comment:
this seems to be a test case for verifying behavior of the *mapTopic* method.
IMHO you should add assertions as to whether the method has actually mapped the message topic to the correct address.

So far we try to reflect the method under test in the test method's name. Maybe something like `testMapTopicSupportsShortAndLongTopicNames`?